lib/sync-get-worker.js

```diff
@@ -5,7 +5,7 @@
 
 const https = require('node:https');
 const workerThreads = require('node:worker_threads');
-const {result} = require('./core');
+const {Result} = require('./core');
```

Comment:
I'd be happy with a `require('./result')` to avoid the evil barrel export (and even so in general). What do you think?

Reply:
I agree! 😀